docker/application/Dockerfile

```diff
@@ -137,6 +137,12 @@ RUN git clone https://github.com/acbecker/hotpants.git \
     && cd .. \
     && rm -rf hotpants
 
+# So, the Debian python now has a thing that tells pip that it really
+# shouldn't be mucking about with the directories that apt manages.
+# This sounds like a really good idea!  Duelling package managers are a
+# nightmare!  But... it breaks the dockerfile.  So, work around it.
+RUN rm /usr/lib/python3.11/EXTERNALLY-MANAGED
```

Comment:
Did.  I'm not sure that separating package managers the way they want to be separated makes sense inside a Dockerfile.  What they do is set up their own clean environment so they can control everything that's there.  Pip does (I think) try to use existing system stuff as much as possible.  But, a Dockerfile is *already* a clean environment.  We don't have to worry about mucking up the system environment for other things, because the docker image exists only for this one thing.  I'd probably want to evaluate how much stuff got redundantly installed when separating the package managers to think about whether it's worth it.